ledger-core/v2/configuration/configuration.go

```diff
@@ -7,14 +7,13 @@ package configuration
 
 import (
 	"fmt"
-	"path/filepath"
-	"reflect"
-	"strings"
+	"github.com/insolar/insconfig"
```

Comment:
wrond imports order

Reply:
goimport file watcher said it's correct.
Don't argue with robot